cpp/include/raft/handle.hpp

```diff
@@ -153,6 +137,19 @@ class handle_t {
     return cusparse_handle_;
   }
 
+  rmm::exec_policy get_thrust_policy() const {
+    std::lock_guard<std::mutex> _(mutex_);
+    if (!thrust_policy_initialized_) {
+      thrust_policy_ = new rmm::exec_policy(get_stream());
+      thrust_policy_initialized_ = true;
+    }
+    return *thrust_policy_;
+  }
+
+  rmm::exec_policy get_thrust_policy(cudaStream_t stream) const {
```

Comment:
Let's drop this function. User should have a clear intent when wanting a thrust policy with a stream different to one on the handle